libs/dh/market-participant/actors/shell/src/lib/dh-market-participant-actors-shell.component.ts

```diff
@@ -17,38 +17,31 @@
 import { Component } from '@angular/core';
 import { TranslocoDirective } from '@ngneat/transloco';
 
-import { WATT_TABS } from '@energinet-datahub/watt/tabs';
-import { DhMarketRolesOverviewComponent } from '@energinet-datahub/dh/market-participant/actors/feature-market-roles';
-import { DhActorsOverviewComponent } from '@energinet-datahub/dh/market-participant/actors/feature-actors';
-import { DhOrganizationsOverviewComponent } from '@energinet-datahub/dh/market-participant/actors/feature-organizations';
+import { WATT_LINK_TABS } from '@energinet-datahub/watt/tabs';
 
 @Component({
   selector: 'dh-market-participant-actors-shell',
   standalone: true,
+  styles: `
+    dh-market-participant-actors-shell {
```

Comment:
But I removed view encapsulation?

Reply:
Hmmm ... maybe it's not committed/pushed yet :)

Reply:
No, the styling applied to `dh-market-participant-actors-shell` selector has no effect because the component has encapsulation enabled. That's why my suggestion was to use `:host` instead.

Reply:
Now i understand!